### PR TITLE
Add method in which exception was found

### PR DESCRIPTION
**Pull request by danmosemsft**
@terrajobst

### CODE REVIEW

File: src/Terrajobst.PlatformCompat.Scanner/ExceptionScanner.cs [hunk 1]
```diff
@@ -87,14 +87,14 @@ private static ExceptionInfo ScanPlatformNotSupported(IMethodDefinition method,
                     op.Value is IMethodReference m &&
                     IsPlatformNotSupported(m))
                 {
-                    return ExceptionInfo.ThrowsAt(nestingLevel);
+                    return ExceptionInfo.ThrowsAt(nestingLevel, method.ToString());
```
Comment (terrajobst) [inline src/Terrajobst.PlatformCompat.Scanner/ExceptionScanner.cs]: I wouldn't use `ToString()`. For consistency, we should use `DocId()`.

File: src/Terrajobst.PlatformCompat.Scanner/ExceptionScanner.cs [hunk 2]
```diff
@@ -87,14 +87,14 @@ private static ExceptionInfo ScanPlatformNotSupported(IMethodDefinition method,
...
                 }
 
                 // throw SomeFactoryForPlatformNotSupportedExeption(...);
                 if (op.Value is IMethodReference r &&
                     IsFactoryForPlatformNotSupported(r))
                 {
-                    return ExceptionInfo.ThrowsAt(nestingLevel);
+                    return ExceptionInfo.ThrowsAt(nestingLevel, method.ToString());
```
Comment (terrajobst) [inline src/Terrajobst.PlatformCompat.Scanner/ExceptionScanner.cs]: Same

File: src/ex-gen/Program.cs [hunk 3]
```diff
@@ -200,6 +200,7 @@ private static void ExportCsv(Database database, string path)
                 writer.Write("Namespace");
                 writer.Write("Type");
                 writer.Write("Member");
+                writer.Write("Site");
```
Comment (terrajobst) [inline src/ex-gen/Program.cs]: I've no problem with emitting the data here, but we should have a global readonly field that controls this, defaulted to `false`.

The reason being that the file `etc\Exception.csv` that is written by this tool will be embedded as a resource in my analyzer. Adding the site the exception occurs resulted in a 60% increase of the file, but the data isn't useful to the analyzer.

**Comment by danmosemsft** [comment]
@krwq too late now, but this is teh change I mentioned that is helpful when analyzing PNSE

**Comment by pjanotti** [comment]
I will pick this up: we probably want something to generate a kind of raw (or full) PNSE with some this and perhaps some extra columns but those should not be included in the csv consumed by the analyzer (this is actually already handled on my current PR - in a very simple way).

**Comment by danmosemsft** [comment]
It's all yours @pjanotti ...

**Comment by pjanotti** [comment]
Implementing it on #45

**Pull request closed without merging**